pkg/scheduler/scheduler.go

```diff
@@ -419,6 +419,8 @@ func (sched *Scheduler) finishBinding(prof *profile.Profile, assumed *v1.Pod, ta
 // scheduleOne does the entire scheduling workflow for a single pod.  It is serialized on the scheduling algorithm's host fitting.
 func (sched *Scheduler) scheduleOne(ctx context.Context) {
 	podInfo := sched.NextPod()
+	podPriority := podutil.GetPodPriority(podInfo.Pod)
+	metrics.PodPriority.Observe(float64(podPriority))
```

Comment:
do we want to record the priority for the same pod if it goes through multiple attempts in multiple scheduling cycles? that seems counter-intuitive.